### PR TITLE
Add formula for 2.4.6

### PR DESCRIPTION
**Pull request by dan-blanchard**

### CODE REVIEW

File: Formula/apache-spark@2.4.6.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  homepage "https://spark.apache.org/"
+  url "https://www.apache.org/dyn/closer.lua?path=spark/spark-2.4.6/spark-2.4.6-bin-hadoop2.7.tgz"
+  mirror "https://archive.apache.org/dist/spark/spark-2.4.6/spark-2.4.6-bin-hadoop2.7.tgz"
+  version "2.4.6"
+  sha256 "7174fee30057fbf226698ebfcfa7c2944a26197e149053aa29b26b250d1e1eba"
+  head "https://github.com/apache/spark.git"
+
+  bottle :unneeded
+
+  depends_on :java => "1.8"
```
Comment (eddies) [inline Formula/apache-spark@2.4.6.rb]: Should now be `depends_on "openjdk@8"` (see https://github.com/eddies/homebrew-spark-tap/pull/3)

**Review by eddies — CHANGES_REQUESTED** [review]
@dan-blanchard Sorry for not getting to this sooner! If you're up for making the change to `depends_on` and squashing down to a single commit, that would be great!

**Comment by dan-blanchard** [comment]
All squashed and good to go

**Review by eddies — APPROVED** [review]

**Comment by eddies** [comment]
> All squashed and good to go

Thanks!

**Pull request merged**